src/Form.js

```diff
@@ -148,10 +158,24 @@ handleChangeAuth =(text)=> {
           <TextBox basicauth='Basic Authorization' bearertoken='Bearer Token' name='textbox' value={this.state.textbox} handler={this.handleChange} authhandle={this.handleChangeAuth} handleauth={this.auth}/>
 
           <br />
-          <Display basicauth='Basic Authorization' data={this.state.data} />
+          <Display basicauth='Basic Authorization' data={this.props.method} />
```

Comment:
One more thing: now that we have Redux (I would have expected the same using Context), I would expect `Display` to `connect` itself to get `data` rather than continuing to be called within `<Form />`. This shared application state should never have stayed inside `Form`.